src/less/designguide.less

```diff
@@ -65,32 +65,25 @@
         font-size: 87.5%;
     }
 
-    .color-container {
-        display: flex;
-        justify-content: center;
+    .color-preview {
+        display: grid;
+        gap: 20px;
```

Comment:
What do you think about doubling the gap? Any reason for the change over to using a lot of px instead of rem?